Classes/Views/NavigationTitleDropdownView.swift

```diff
@@ -0,0 +1,121 @@
+//
```

Comment:
Is this guy accessible at this moment? I don't think it's being recognised as a `Button` now.

Reply:
Crap! Nope, it needs some AX love for sure.

Reply:
Want to take it on now or should I create a ticket?

Reply:
@BasThomas mind filing an issue?